addons/product/views/product_template_views.xml

```diff
@@ -15,7 +15,7 @@
                 <field name="priority" widget="priority" optional="show" nolabel="1"/>
                 <field name="name" string="Product Name"/>
                 <field name="default_code" optional="show"/>
-                <field name="product_tag_ids" widget="many2many_tags" options="{'color_field': 'color'}" optional="show"/>
+                <field name="product_tag_ids" string="Tags" widget="many2many_tags" options="{'color_field': 'color'}" optional="show"/>
```

Comment:
I found it confusing to have tags on product template list view but not on the product product list views